graphql_compiler/schema.py

```diff
@@ -267,6 +267,14 @@ def _parse_datetime_value(value):
     parse_literal=_unused_function,  # We don't yet support parsing Decimal objects in literals.
 )
 
+GraphQLAny = GraphQLScalarType(
```

Comment:
We supported the Any OrientDB type in the SchemaGraph so I created this GraphQLType so that we didn't break the support of previously supported types. The other option would be just to ignore the Any OrientDB type and throw a warning. I am not particularly inclined for either choice though.

Reply:
I will remove it in the next pr. I might add it again later if we deem it crucial to support OrientDB's Any type.

Reply:
I think we don't need it for now, and if we do, we'll have to come up with a good way of representing it. But that's work for another time -- let's cut everything we aren't sure we need, and try to get the schema graph stuff merged quickly.

Reply:
Ok makes sense.

Reply:
Removed in https://github.com/kensho-technologies/graphql-compiler/pull/269